### PR TITLE
Update README.md

### LINKED ISSUE #2: Add link to IaC-GCP repository
The readme page currently links the Azure and AWS projects.
Add the link to the GCP project: 
[SAS Viya 4 Infrastructure as Code (IaC) for Google Cloud Platform (GCP)](https://github.com/sassoftware/viya4-iac-gcp )

### PR DESCRIPTION
**Pull request by kevinlinglesas**
Add reference to new repo https://github.com/sassoftware/viya4-iac-gcp supporting IaC for Google Cloud Platform (GCP)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ The [SAS Viya Orders CLI](https://github.com/sassoftware/viya4-orders-cli) can b
 The SAS Viya Infrastruture as Code repositories can be used to create the necessary compute resources for a SAS Viya deployment. The following cloud provider is supported at this time:
 - [SAS Viya Infrastructure as Code - Azure](https://github.com/sassoftware/viya4-iac-azure)
 - [SAS Viya Infrastructure as Code - AWS](https://github.com/sassoftware/viya4-iac-aws)
+- [SAS Viya Infrastructure as Code - Google Cloud Platform (GCP)](https://github.com/sassoftware/viya4-iac-gcp)
```
Comment (thpang) [inline README.md]: Remove `Google Cloud Platform` from the text. GCP is understood.

**Review by thpang — CHANGES_REQUESTED** [review]
If you can review comment. Thx.

File: README.md (hunk 1 above)
Comment (kevinlinglesas) [reply README.md]: Great.  That's really how I wanted to write it.  I was compromising between abbreviation and existing pattern of the README and what issue #2 submitter had offered.

**Review by thpang — APPROVED** [review]
Looks good.

**Pull request merged**